src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/ConversionCompletionProvider.cs

```diff
@@ -77,6 +77,12 @@ private void AddUserDefinedConversionsOfType(ArrayBuilder<CompletionItem> builde
                                              // Lifted conversion https://docs.microsoft.com/hu-hu/dotnet/csharp/language-reference/language-specification/conversions#lifted-conversion-operators
                                              select CreateSymbolCompletionItem(m, typeName, targetTypeIsNullable: containerIsNullable && m.ReturnType.IsStructType(), position);
                 builder.AddRange(allExplicitConversions);
+
+                // Base types are valid sources for user-defined conversions
+                // https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/conversions#processing-of-user-defined-explicit-conversions
+                // "Find the set of types, D, from which user-defined conversion operators will be considered. This set consists of S0 (if S0 is a class or struct), 
+                // the base classes of S0 (if S0 is a class), T0 (if T0 is a class or struct), and the base classes of T0 (if T0 is a class)."
+                // Note: We only look in the source (aka container), because target could be any type (in scope) of the compilation.
```

Comment:
```suggestion
                // Base types are valid sources for user-defined conversions
                // Note: We only look in the source (aka container), because target could be any type (in scope) of the compilation.
```

Reply:
spec'ese is rarely necessary except for extremely complex scenarios.  it's totally suffiicent to say that conversions may be found in base types.